examples/js/controls/OrbitControls.js

```diff
@@ -1,3 +1,5 @@
+console.warn( 'THREE example script "js/controls/OrbitControls.js" is deprecated and will soon be removed. Use the ES module version instead: "jsm/controls/OrbitControls.js".' );
```

Comment:
I would include the intended date of removal, to force feedback sooner than later, and perhaps reference an issue. Suggestion:

> THREE.OrbitControls: Scripts in "examples/js/*" are deprecated and will be removed in January 2020 (r113). Please use the ES module instead, "examples/jsm/controls/OrbitControls.js". See: `https://github.com/mrdoob/three.js/pull/<TBD>`.

^Let's open a new issue to track the change and help users resolve issues.

Reply:
Sounds good I'll make the change -- @mrdoob it should probably be you who makes the issue to announce the intent to remove and then I can reference that?

Reply:
Ping @mrdoob any plans to make an issue with a deprecation date?

Reply:
@mrdoob any updated plans on this?

Reply:
AFAIK, no files depend in the repository depend on `examples/js` anymore. However, it would be good if an additional pair of eyes verifies this^^.

If there are no dependencies anymore, I think we should move on with the removal by adding this log to all js files.

Reply:
We should also publish a small guide in the docs (maybe an additional section in [Import via modules](https://threejs.org/docs/index.html#manual/en/introduction/Import-via-modules)) which explains what users have to do if they used this style:
```js
const THREE = window.THREE = require( 'three' );
require( 'three/examples/js/loaders/GLTFLoader.js' );
```
or a included the scripts directly:
```js
<script src="https://unpkg.com/three@0.112.0/build/three.js"></script>
<script src="https://unpkg.com/three@0.112.0/examples/js/loaders/GLTFLoader.js"></script>
```
This would make support easier since we can always refer to the same guide.